Jenkinsfile.mom.si

```diff
@@ -47,6 +47,7 @@ node('shakedown') {
                """
           } finally {
             junit allowEmptyResults: true, testResults: 'shakedown.xml'
+            archive includes: "diagnostics.zip"
```

Comment:
Here shakedown is called directly (not `si_pipeline`) - the bundle will not be created. Besides I don't think Jenkinsfile is used anywhere.